README.md

```diff
@@ -1,75 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Getting Started with Create React App
```

Comment:
Great job @NurkaAmre 🤝

- [ ] [Optional] Kindly use the correct readme file provided in the project requirements and customize it accordingly. You may use this [readme template](https://github.com/microverseinc/readme-template) to get started.

Reply:
@1GORDON thank you very much 😄